frontend-dashboard/src/components/ControlSlider.jsx

```diff
@@ -1,9 +1,26 @@
-export default function ControlSlider({ name }) {
+import useSocket from "../hooks/useSocket";
+
+export default function ControlSlider({ name, min, max, step, defaultValue }) {
+  const { sendMessage } = useSocket();
+
+  const handleSliderChange = (e) => {
+    const sliderValue = e.target.value;
```

Comment:
```suggestion
    const sliderValue = Number(e.target.value);
```
검색해보니 e.target.value는 문자열 형태로 들어온다는 것을 확인했습니다. 실제로 typeof를 사용해 확인해본 결과, 값의 타입이 "string"인 것을 확인할 수 있었습니다.

하지만 우리가 사용하려는 데이터 타입은 숫자이기 때문에, 이 값을 다른 곳에서 그대로 사용할 경우 예상치 못한 오류가 발생할 수 있을 것 같습니다. 이를 방지하기 위해 Number()를 사용하여 숫자로 변환해주는 것이 어떨까 싶습니다!